.github/pull_request_template.md

```diff
@@ -0,0 +1,45 @@
+### This pull request closes:
+
+✔️ Closes: #<!-- Insert issue number here -->
+🏁 Jira Issue: [TC-<!-- Insert issue number here -->](<!-- Insert issue link here -->)
+
+### Which service(s) does this issue affect:
+
+- [ ] 🛰️ API
+- [ ] 🛒 webstore
+- [ ] 🎓 admin-page
+
+### Types of changes
+
+- [ ] 🐛 Bug fix (a non-breaking change which fixes an issue)
```

Comment:
We have other types of changes like ci, refactor etc
This pr is not adding a new feature, it is either a chore or docs

I would add an extra option called `other`

Reply:
It is literally a new feature, but I get what you mean.